build.gradle

```diff
@@ -34,6 +33,8 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.4'
     id 'com.dorongold.task-tree' version '1.3'
     id 'org.ajoberstar.grgit' version '1.7.1' apply false
+    id 'net.ltgt.errorprone-javacplugin' version '0.3' apply false
+    id "com.github.spotbugs" version "1.6.2"
```

Comment:
We want to stick on 4.9 for the time being, since 4.10 clears away cached Gradle wrappers and breaks offline builds for FRC team projects.

But I'll be glad to update the spotbugs version